src/20_Components/amp-bind.html

```diff
@@ -158,7 +165,19 @@
       `amp-state` components
     -->
     <button class="ampstart-btn caps m1"
-            on="tap:AMP.setState({currentAnimal: 'cat'})">Set to Cat</button>
-  </section>
+            on="tap:AMP.setState({allAnimals: {currentAnimal: 'cat'}})">
+                                  Set to Cat
+    </button>
+
+    <!-- ## Load state remotely -->
+    <!--
+      By setting the src value of the `amp-state`, new data are fetched remotely and merged into the `amp-state` data json.
```

Comment:
s/src value/`src` attribute/ (including the code formatting for src)
s/data are/data is/ 
remove 'remotely' and 'data json'  (not needed)